prow/tide/tide.go

```diff
@@ -334,6 +335,18 @@ func (sc *statusController) setStatuses(all, pool []PullRequest) {
 			}
 		}
 		if wantState != strings.ToLower(string(actualState)) || wantDesc != actualDesc {
```

Comment:
Should we check that the URL is what we expect here now too?

Reply:
That would only help the case where we migrate config and would cause Tide to update a lot of statuses -- do we care?

Reply:
Probably not. Lets leave it as is for now and change it later if we find that we want to keep the URLs more strictly up to date.